src/components/distributions/editor/TextForm/TextInput.js

```diff
@@ -6,7 +6,57 @@ import TextArea from 'react-textarea-autosize'
 
 import DistributionSelector from './DistributionSelector'
 
-import insertAtCaret from 'lib/jquery/insertAtCaret'
+import {EditorState, Editor, ContentState, getDefaultKeyBinding, KeyBindingUtil, Modifier} from 'draft-js'
```

Comment:
Put this import up in the section with the other npm modules (just below 'TextArea').
(which, I'm guessing isn't needed anymore, as well? Also, ReactDOM, it appears?)